### PR TITLE
Display db

### PR DESCRIPTION
**Pull request by neilbulloch**

### CODE REVIEW

**Review by olyolyoly — COMMENTED** [review]
Good to go

File: admin/admin.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=Portfolio", 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = 'SELECT `title`, `img_url` FROM `projects`;';
+
+$query = $db->prepare($sql);
+$query->execute();
+
+$result = $query->fetchALL();
```
Comment (mporam) [inline admin/admin.php]: `fetchAll` not `fetchALL`

File: index.php [hunk 2]
```diff
@@ -69,22 +83,15 @@ <h3 class="about_me_header">Do or do not. There is no try</h3>
...
-            </div>
-            <div class="projects_container">
-                <img class="project_img" src="Images/rsz_jumbotronbuild.png" alt="jumbotron build">
 
-                <a class="project_link" title="Link to Hello World page exercise" target="_blank" href="#">Hello World</a>
-            </div>
 
-            <div class="projects_container">
-                <img class="project_img" src="Images/maydenacademylogo.png" alt="mayden logo">
+            <?php foreach ($result as $project) {
```
Comment (mporam) [inline index.php]: what if $result is empty?

**Review by mporam — CHANGES_REQUESTED** [review]

File: index.php [hunk 3]
```diff
@@ -69,22 +82,20 @@ <h3 class="about_me_header">Do or do not. There is no try</h3>
...
+                foreach ($result as $project) {
+                    echo '<div class="projects_container">';
+                    echo '<img class="project_img" src="' . $project['img_url'] . '" alt = "' . $project['title'] . '">' ;
+                    echo '<a class="project_link" href="#">' . $project['title'] . '</a>';
+                    echo '</div>';
+                }
 
-            <div class="projects_container">
-                <img class="project_img" src="Images/maydenacademylogo.png" alt="mayden logo">
+            } else echo '<h1> Database connection failed. Please refresh page. </h1>'
```
Comment (mporam) [inline index.php]: bad neil

File: index.php [hunk 4]
```diff
@@ -1,4 +1,17 @@
-<!doctype html>
```
Comment (mporam) [inline index.php:1]: why? please put back in

**Review by mporam — APPROVED** [review]

**Pull request merged**